tests/kernel/mem_protect/userspace/testcase.yaml

```diff
@@ -1,6 +1,6 @@
 tests:
   kernel.memory_protection.userspace:
-    filter: CONFIG_ARCH_HAS_USERSPACE
+    filter: CONFIG_ARCH_HAS_USERSPACE and CONFIG_SRAM_SIZE > 32
```

Comment:
we have min_ram and min_flash keys that can be used instead of Kconfig variables, please use those.

Reply:
ok